requirements.txt

```diff
@@ -1,5 +1,3 @@
-ansible==6.5.0
-ansible-compat==2.2.1
```

Comment:
These will not break existing tests because https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18133 is already merged, right? Just to confirm.

Reply:
yes, the fact that PR#18133 has been merged in advance result in test not finding the requirements.yaml (as this PR is not already in, or if test are using fixed old version of qe-sap-deployment) and so installing  `ansible==6.5.0` and having everything installed as `ansible==6.5.0` is the old approach with everything inside it.

Once committing this PR, the test will initially not installing `ansible==6.5.0` but the smaller `ansible-core` using pip. But as this same PR also has a new file named `requirements.yaml`, this will result on the test side in `ansible-galaxy install` to be called too, that will result in installing everything that Ansible needs and was previously provided by `ansible==6.5.0`.